front-end/src/components/Header/User/User.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import "./style.scss";
```

Comment:
@beeeeesankun -san
About User component.
I think do can separate out site header. We can re-use it.
Maybe it not belong to Header.

Ex: Avatar Component.
```JS
export default function Avatar(props) {
  const {img, name} = props;
  return (
    <>
      <div className="user">
        <img src={img} alt="" />
        <div className="user-profile">
          <h3>{name}</h3>
          <p>Administrator</p>
        </div>
      </div>
    </>
  );
}

Use it:
<Avatar img="PATH" name="username"/>
```